dao/src/main/resources/db/migration/V83__scannerSummariesIssuesTimestamps.sql

```diff
@@ -0,0 +1,29 @@
+-- This migration script extracts the timestamp from the issues table and moves it up to the scan_summaries_issues
+-- table. This is a preparation to deduplicate the entities in the issues table; the table will store only the
+-- most relevant properties for issues.
+
+-- Since the timestamp column is not nullable, create a new table that is populated and then renamed.
+CREATE TABLE scan_summaries_issues_with_timestamps
+(
+    id                bigserial                         PRIMARY KEY,
+    scan_summary_id   bigint REFERENCES scan_summaries  NOT NULL,
+    issue_id          bigint REFERENCES issues          NOT NULL,
+    timestamp         timestamp                         NOT NULL
+);
+
+INSERT INTO scan_summaries_issues_with_timestamps
```

Comment:
Do we need some indexes on this table ?
Maybe also some foreign key constraints ?

Reply:
Foreign key constraints are in place due to the `REFERENCES` keywords. I am not sure whether we need an index, and if yes, on which columns. Maybe we should monitor the performance in production.

Reply:
With Postgres you generally need an index on the foreign key columns i.e. `scan_summary_id` and `scan_summary_id`.

However since the previous indexes caused performance problem, I agree to can wait to see how it behaves in production.